src/fides/data/privacy_notices/privacy_notice_templates.yml

```
@@ -1,4 +1,400 @@
 privacy_notices:
+### GPP State-by-state ###
+- name: Data sales and sharing
+  notice_key: data_sales_sharing_gpp_us_state
+  id: pri_gpp_1 # TODO: update
+  description: We may transfer or share your personal information to third parties in exchange for monetary or other valuable consideration or for the purposes of [cross-context behavioral targeted advertising]. You can learn more about what information is used for this purpose in our privacy notice.
+  internal_description: Data sales and sharing (GPP, California) # TODO: update?
+  regions:
+    - us_ca
+  consent_mechanism: opt_out
+  data_uses:
+    - marketing.advertising.first_party.targeted
+    - marketing.advertising.third_party.targeted
+  enforcement_level: not_applicable
+  disabled: True
+  has_gpc_flag: False # TODO: not sure?
+  displayed_in_privacy_center: True
+  displayed_in_overlay: True
+  displayed_in_api: False
+  framework: gpp_us_state
+  gpp_field_mapping:
+    - region: us_ca
+      notice:
+        - SaleOptOutNotice
```

I think we need `SharingOptOutNotice` here too
```suggestion
        - SaleOptOutNotice
        - SharingOptOutNotice
```

---

yes good catch, thank yoU!